component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/backupcode/util/BackupCodeUtil.java

```diff
@@ -122,11 +125,12 @@ public static RealmService getRealmService() {
      * @return The userRealm.
      * @throws BackupCodeException If an error occurred while getting the user realm.
      */
-    public static UserRealm getUserRealm(String username) throws BackupCodeException {
+    private static UserRealm getUserRealm(String username) throws BackupCodeException {
```

Comment:
Shall we keep it public, so that we are not breaking any customization

Reply:
@somindatommy WDYT?

Reply:
We have recently introduced this method. Do we really need to have this as a public method? I feel we have added this to make sure that we can easily write the tests. @thanujalk

Reply:
Then it is ok...